registry/storage/driver/azure/parser.go

```diff
@@ -8,7 +8,9 @@ import (
 )
 
 const (
-	defaultRealm = "core.windows.net"
+	defaultRealm                  = "core.windows.net"
+	defaultCopyStatusPollMaxRetry = 10
```

Comment:
I think this default value is a bit too high 🤔 we should change that to 5

Reply:
done.